ccl/changefeedccl: TestChangefeedSchemaChangeBackfillCheckpoint failed

ccl/changefeedccl.TestChangefeedSchemaChangeBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14072188?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/14072188?buildTab=artifacts#/) on release-23.1 @ [b102a6dd3f49ef7bb92217d005ad890105f84d07](https://github.com/cockroachdb/cockroach/commits/b102a6dd3f49ef7bb92217d005ad890105f84d07):


```
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedSchemaChangeBackfillCheckpoint3530097046
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedSchemaChangeBackfillCheckpoint
    changefeed_test.go:2511: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedSchemaChangeBackfillCheckpoint3530097046
--- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint (51.43s)
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint/cloudstorage
    helpers_test.go:823: making server as secondary tenant
    helpers_test.go:901: making cloudstorage feed factory
    changefeed_test.go:2402: condition failed to evaluate within 45s: waiting for checkpoint
    --- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint/cloudstorage (51.41s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedSchemaChangeBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/14090899?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_UnitTests_BazelUnitTests/14090899?buildTab=artifacts#/) on release-23.1 @ [333253387f07fcbfc20697df5556224d44ca86c2](https://github.com/cockroachdb/cockroach/commits/333253387f07fcbfc20697df5556224d44ca86c2):


```
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/b239b1bb590a7eed1fa5a8194849ef12/logTestChangefeedSchemaChangeBackfillCheckpoint3595143624
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedSchemaChangeBackfillCheckpoint
    changefeed_test.go:2511: -- test log scope end --
--- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint (53.40s)
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint/pubsub
    helpers_test.go:823: making server as secondary tenant
    helpers_test.go:901: making pubsub feed factory
    changefeed_test.go:2402: condition failed to evaluate within 45s: waiting for checkpoint
    --- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint/pubsub (53.37s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/changefeedccl.TestChangefeedSchemaChangeBackfillCheckpoint [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/15479277?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/15479277?buildTab=artifacts#/) on release-23.1 @ [bb0d40a1c8334ab6584189c004f358c63bd95aa7](https://github.com/cockroachdb/cockroach/commits/bb0d40a1c8334ab6584189c004f358c63bd95aa7):


```
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedSchemaChangeBackfillCheckpoint3391357835
    test_log_scope.go:79: use -show-logs to present logs inline
=== CONT  TestChangefeedSchemaChangeBackfillCheckpoint
    changefeed_test.go:2511: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/905465f697dc04c39f7d8a82cb057bfe/logTestChangefeedSchemaChangeBackfillCheckpoint3391357835
--- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint (51.32s)
=== RUN   TestChangefeedSchemaChangeBackfillCheckpoint/enterprise
    helpers_test.go:823: making server as secondary tenant
    helpers_test.go:901: making enterprise feed factory
    helpers_test.go:915: pgURL enterprise root
    changefeed_test.go:2402: condition failed to evaluate within 45s: waiting for checkpoint
    --- FAIL: TestChangefeedSchemaChangeBackfillCheckpoint/enterprise (51.31s)
```

Parameters:
 - <code>TAGS=bazel,gss</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestChangefeedSchemaChangeBackfillCheckpoint.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/changefeedccl/changefeed_test.go
+++ b/pkg/ccl/changefeedccl/changefeed_test.go
@@ -2299,7 +2299,7 @@ func TestChangefeedSchemaChangeBackfillCheckpoint(t *testing.T) {
 		}
 
 		// Setup changefeed job details, avoid relying on initial scan functionality
-		baseFeed := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved='100ms', no_initial_scan`)
+		baseFeed := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved='100ms', min_checkpoint_frequency='100ms', no_initial_scan`)
 		jobFeed := baseFeed.(cdctest.EnterpriseTestFeed)
 		jobRegistry := s.Server.JobRegistry().(*jobs.Registry)
 
@@ -2330,6 +2330,7 @@ func TestChangefeedSchemaChangeBackfillCheckpoint(t *testing.T) {
 		testutils.SucceedsSoon(t, func() error {
 			prog := loadProgress()
 			if p := prog.GetHighWater(); p != nil && !p.IsEmpty() {
+				t.Logf("highwater: %s", p)
 				return nil
 			}
 			return errors.New("waiting for highwater")

